playground/src/import-url/import-url.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+require('./import-url.scss');
+
+import events from '../events/events.js';
+const fetch = require('node-fetch');
+import * as Button from '../button/button.js';
+import FlyIn from '../fly-in/base.js';
+
+export const EVENT_NEW_URL_INPUT =
+  'event-new-url-input';
+
+
+export function createImportURLView(target, trigger) {
+  return new ImportURL(target, trigger);
+}
+
+class ImportURL extends FlyIn {
+  constructor(target, trigger) {
+    super(target);
+
+    this.target = target;
+    this.trigger = Button.from(trigger, this.toggle.bind(this));
+
+    const content = document.createElement('div');
+    content.className = 'import-url';
+    content.innerHTML = `
+      <p class="experimental-view-intro">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris. Learn more about experimental features.</p>
+      <form id="url-bar" class="import-url-bar">
+        <input id="url-bar-input" class="import-url-bar-input" placeholder="Your URL" type="url" required pattern="https?://.+" name="input">
```

Comment:
This needs to have `font-family: Noto`:

![Bildschirmfoto 2020-06-18 um 10 19 51](https://user-images.githubusercontent.com/12857772/85002646-7155b400-b155-11ea-83a2-9c77f5c16a19.png)

We should also make this accept URLs without a protocol. In general in my opinion this should provide almost the same UX as the browsers URL bar.

Reply:
Alternatively the input should have a label explaining the expected scheme.